SoundSwitch/UI/Forms/Settings.cs

```diff
@@ -824,5 +846,77 @@ private void UsePrimaryScreenCheckbox_CheckedChanged(object sender, EventArgs e)
         }
 
         #endregion
+
+        #region Troubleshooting
+
+        private void ResetAudioDevicesButton_Click(object sender, EventArgs e)
+        {
+            AudioSwitcher.Instance.ResetProcessDeviceConfiguration();
+        }
+
+        private void ExportLogFilesButton_Click(object sender, EventArgs e)
+        {
+            var saveFileDialog = new SaveFileDialog
+            {
+                InitialDirectory = "C://",
```

Comment:
Better to leave this unset. Not everybody has a C: drive. Or use that as their main drive. 

If empty, I think it opens in the last place the user went with explorer